pennylane/devices/default_qutrit.py

```diff
@@ -205,9 +209,15 @@ def _apply_operation(self, state, operation):
             return state
         wires = operation.wires
 
-        if operation.name in self._apply_ops:
+        if operation.name in self._apply_ops:  # pylint: disable=no-else-return
             axes = self.wires.indices(wires)
             return self._apply_ops[operation.name](state, axes)
+        elif (
+            isinstance(operation, qml.ops.Adjoint)  # pylint: disable=no-member
+            and operation.base.name in self._apply_ops
```

Comment:
perhaps we should use the top-level name since you did register it? unless you want to inadvertently support other ops that make it this far, though I feel like they won't make it this far usually.
```suggestion
            and operation.name in self._apply_ops
```

Reply:
This won't work as the `inverse` kwarg of the `_apply_ops` method needs to be set for adjoint operators. As far as making it this far is concerned, that is not a concern as `Device.check_validity` would raise and error for unsupported ops, including unsupported adjoint ops.